front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/AddPathStepPopup.tsx

```diff
@@ -23,8 +23,8 @@ import type { FeatureInfoClick } from '../types';
 
 type AddPathStepPopupProps = {
   pathProperties?: ManageTrainSchedulePathProperties;
-  featureInfoClick: FeatureInfoClick;
-  setFeatureInfoClick: React.Dispatch<React.SetStateAction<FeatureInfoClick>>;
+  featureInfoClick: FeatureInfoClick | undefined;
+  setFeatureInfoClick: React.Dispatch<React.SetStateAction<FeatureInfoClick | undefined>>;
```

Comment:
To simplify the code, you can even only pass a function `resetFeatureInfoClick` here instead of `setFeatureInfoClick`
```suggestion
  resetFeatureInfoClick: () => void;
```

Reply:
Done